app/DataTransferObjects/LicenseManagement/LicenseLimitsData.php

```diff
@@ -51,6 +51,9 @@ public function __construct(
         public string $resetDate,
     ) {}
 
+    /**
+     * @return array
```

Comment:
This static fix isn't complete

```suggestion
     * @return array<string, string>
```